engine/flink/kafka-components-utils/src/main/scala/pl/touk/nussknacker/engine/kafka/source/flink/FlinkKafkaSource.scala

```diff
@@ -43,11 +43,11 @@ import java.time.Duration
 import java.util.Properties
 import scala.jdk.CollectionConverters._
 
-class FlinkKafkaSource[T](
+abstract class FlinkKafkaSource[T](
```

Comment:
I did and experiment with making this class abstract and removing `timestampAssignerForTest` and `typeInformation` because it is not instantiated directly anywhere - only as `FlinkConsumerRecordBasedKafkaSource` which overrides these. The tests are passing so I think they were never used.